modules/vidazooBidAdapter.js

```diff
@@ -0,0 +1,116 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+import { BANNER } from 'src/mediaTypes';
+
+const BIDDER_CODE = 'vidazoo';
+const CURRENCY = 'USD';
+const TTL_SECONDS = 60 * 5;
+export const URL = '//prebid.cliipa.com';
+// export const URL = '//localhost:8067';
+const INTERNAL_SYNC_TYPE = {
+  IFRAME: 'iframe',
+  IMAGE: 'img'
+};
+const EXTERNAL_SYNC_TYPE = {
+  IFRAME: 'iframe',
+  IMAGE: 'image'
+};
+
+function isBidRequestValid(bid) {
+  const params = bid.params || {};
+  return !!(params.cId && params.pId);
+}
+
+function buildRequest(bid, topWindowUrl, size) {
+  const {params, bidId} = bid;
+  const {bidFloor, cId, pId} = params;
+
+  return {
+    method: 'GET',
+    url: `${URL}/prebid/${cId}`,
+    data: {
+      width: size[0],
```

Comment:
this may break for double size arrays (ie `[[300,250]]` ). See https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L113 for help parsing. You can use this function.